### PR TITLE
libvirt_hooks: Modify interface type before test.

### PR DESCRIPTION
**Pull request by rbian**
Network hook only for interface with network type, reset the domain
xml before the test.

### CODE REVIEW

File: libvirt/tests/src/libvirt_hooks.py [hunk 1]
```diff
@@ -255,10 +255,25 @@ def attach_hook():
...
+        vmxml.add_device(iface_xml)
+        vmxml.sync()
+
     def network_hook():
         """
         Check network hooks.
         """
+        # Set interface to use default network
+        edit_iface()
         net_name = "default"
```
Comment (will-Do) [inline libvirt/tests/src/libvirt_hooks.py]: Maybe it's better to make the net_name configurable, then pass it to the edit func, what do u think?

Comment (rbian) [inline libvirt/tests/src/libvirt_hooks.py]: Currently we always use 'default' network for testing. It's okay to make it configurable. Thanks.

**Comment by will-Do** [comment]
It's better to add 'net_name' to cfg too, that's the reason we want it configurable. Thanks.

**Comment by rbian** [comment]
Thanks @will-Do , updated the code.

**Comment by will-Do** [comment]
ACK and merge it, thanks

**Pull request merged**